dts/bindings/clock/st,stm32-rcc.yaml

```diff
@@ -58,12 +58,22 @@ properties:
       required: false
       enum:
         - 1
+        - 2
+        - 4
+        - 8
+        - 16
+        - 32
+        - 64
+        - 128
+        - 256
+        - 512
       description: |
           AHB prescaler. Sets a HCLK frequency (feeding Cortex-M Systick)
           lower than SYSCLK frequency (actual core frequency).
           Zephyr doesn't make a difference today between these two clocks.
```

Comment:
"Zephyr doesn't make a difference today between these two clocks."
Let's remove this sentence now, as it's not valid anymore.

Reply:
removed